src/main/scala/com/github/tototoshi/csv/CSVReader.scala

```diff
@@ -123,7 +123,7 @@ object CSVReader {
     open(file, this.DEFAULT_ENCODING)(format)
   }
 
-  def open(file: File, encoding: String)(implicit format: CSVFormat): CSVReader = {
```

Comment:
For compatibility, please add new methods with `Charset` instead of replacing this.